catalog/spatial/csw/spatial-csw-endpoint/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/event/SendEventTest.java

```diff
@@ -234,6 +236,7 @@ public void testIsAvailableSubjectExperation() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
Is there a comment we can put in as to why this is ignored? just thinking of when it gets picked back up or does that no make any sense?